.github/workflows/jetbrains-updates.yml

```diff
@@ -46,3 +46,23 @@ jobs:
       exampleRepo: https://github.com/gitpod-io/template-php-laravel-mysql
     secrets:
       slackWebhook: ${{ secrets.IDE_SLACK_WEBHOOK }}
+  rubymine:
+    uses: ./.github/workflows/jetbrains-updates-template.yml
+    with:
+      productName: RubyMine
+      productId: rubymine
+      productCode: RM
+      productType: release
+      exampleRepo: https://github.com/gitpod-io/template-jetbrains-rubymine
+    secrets:
+      slackWebhook: ${{ secrets.IDE_SLACK_WEBHOOK }}
+  webstorm:
+    uses: ./.github/workflows/jetbrains-updates-template.yml
+    with:
+      productName: WebStorm
+      productId: webstorm
+      productCode: WS
+      productType: release
+      exampleRepo: https://github.com/gitpod-io/template-jetbrains-webstorm
```

Comment:
We need to review/unarchive these two example repos:
- https://github.com/gitpod-io/template-jetbrains-rubymine
- https://github.com/gitpod-io/template-jetbrains-webstorm

/cc @loujaybee

Reply:
Alternatively, we could use the following up-to-date templates for:
- RubyMine: https://github.com/gitpod-io/template-ruby-on-rails-postgres
- WebStorm: https://github.com/gitpod-io/template-typescript-react

Reply:
Yep, will take a look at them, shouldn't block this PR, though, right? 

CC: @Siddhant-K-code - let's add this to the backlog to review.

Reply:
I think.

For webstorm, [`template-typescript-react`](https://github.com/gitpod-io/template-typescript-react) is good & easy repo to test it out.